projects/goharbor/harbor/docker/linux/harbor-chartserver/Dockerfile

```diff
@@ -0,0 +1,33 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG RELEASE_BRANCH
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install -y shadow-utils >> /dev/null \
+    && yum clean all \
+    && groupadd -f -r -g 10000 chart \
+    && useradd --no-log-init -m -g 10000 -u 10000 chart
+
+COPY --chown chart:chart _output/$RELEASE_BRANCH/dependencies/$TARGETOS-$TARGETARCH/eksa/helm/chartmuseum/chartm /home/chart
+COPY --chown chart:chart _output/make/photon/chartserver/docker-entrypoint.sh /home/chart/
+COPY --chown chart:chart _output/make/photon/common/install_cert.sh /home/chart/
+COPY --chown chart:chart _output/LICENSES /LICENSES
+COPY --chown chart:chart ATTRIBUTION.txt /ATTRIBUTION.txt
+
+RUN chown -R chart:chart /etc/pki/tls/certs \
+    && chown -R chart:chart /home/chart \
```

Comment:
I'm not sure i see the need for these 4 lines, do the scripts not already have the +x flag? isnt the home/chart dir already owned by chart from the copy call?  

Similar question to the certs dir, is not world readable?